src/Utils/BlockTemplateUtils.php

```diff
@@ -254,15 +254,12 @@ public static function convert_slug_to_title( $template_slug ) {
 	 * Converts template paths into a slug
 	 *
 	 * @param string $path The template's path.
-	 * @param string $directory_name The template's directory name.
 	 * @return string slug
 	 */
-	public static function generate_template_slug_from_path( $path, $directory_name = 'block-templates' ) {
-		return substr(
-			$path,
-			strpos( $path, $directory_name . DIRECTORY_SEPARATOR ) + 1 + strlen( $directory_name ),
-			-5
-		);
+	public static function generate_template_slug_from_path( $path ) {
+		$template_extension = '.html';
+
+		return basename( $path, $template_extension );
```

Comment:
Much better. :clap:

Reply:
Thanks! It was also necessary, by the way. I didn't simplify it just for the sake of it. But because our dir structure became `templates/templates`, the original `substr` code would not work anymore.